src/lib/QtWidgets/QTreeWidget.ts

```diff
@@ -83,7 +83,6 @@ export class QTreeWidget extends QAbstractScrollArea<QTreeWidgetSignals> {
     }
 
     insertTopLevelItem(index: number, item: QTreeWidgetItem): void {
-        this.topLevelItems.add(item);
```

Comment:
If you remove this. Wont the items be garbage collected?

Reply:
Oops, didn't realize I removed that.